internal/appfunction/context.go

```diff
@@ -60,6 +62,7 @@ type Context struct {
 	retryData           []byte
 	responseContentType string
 	contextData         map[string]string
+	topicPlaceholders   *regexp.Regexp
```

Comment:
do we want to rename this to valuePlaceholderSpec since it is now beyond Topics?

Reply:
Yes good catch